quoridor-logic-server/src/main/java/com/rl/mpquoridor/models/Pawn.java

```diff
@@ -0,0 +1,111 @@
+package com.rl.mpquoridor.models;
+
+import com.rl.mpquoridor.enums.Direction;
+import lombok.Data;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+@Data
+public class Pawn {
+    private int id;
+    private String color;
```

Comment:
Color should not be in server side.
Only in client side.

Reply:
Like guy said